app/controllers/multifactor_auths_controller.rb

```diff
@@ -8,15 +8,23 @@ class MultifactorAuthsController < ApplicationController
   def new
     @seed = ROTP::Base32.random_base32
     session[:mfa_seed] = @seed
+    session[:mfa_seed_expire] = (Time.now.utc + 30.minutes).to_i
```

Comment:
What about at least "constantize" this time range or better moving this to some config file?

Reply:
Yes. I plan to put it in 'config/application.rb'.